src/box/errcode.h

```diff
@@ -304,8 +304,9 @@ struct errcode_record {
 	/*249 */_(ER_UNKNOWN_AUTH_METHOD,	"Unknown authentication method '%s'") \
 	/*250 */_(ER_INVALID_AUTH_DATA,		"Invalid '%s' data: %s") \
 	/*251 */_(ER_INVALID_AUTH_REQUEST,	"Invalid '%s' request: %s") \
-	/*252 */_(ER_WEAK_PASSWORD,		"Password doesn't meet security requirements: %s") \
-
+	/*252 */_(ER_WEAK_PASSWORD,		"Password doesn't meet security requirements: %s")                                                  \
+	/*253 */_(ER_NO_SUCH_SESSION,		"Session %" PRIu64 " does not exist") \
```

Comment:
Nit: We don't use `PRIxxx` macros in this file. Actually, we try not to use them at all unless absolutely necessary (like it was the case with backtrace format), because they look ugly in the code. Instead, we convert formatted values to standard C types:

```C
/*253 */_(ER_NO_SUCH_SESSION,		"Session %llu does not exist") \

diag_set(ClientError, ER_NO_SUCH_SESSION, (unsigned long long)id);
```

Reply:
> Actually, we try not to use them at all unless absolutely necessary (like it was the case with backtrace format), because they look ugly in the code. Instead, we convert formatted values to standard C types

As you've probably figured out by now, I don't like unnecessary conversions, so I prefer a portable approach with `PR_` macros. I am interested in @alyapunov's opinion.

Reply:
Discussed this matter with @locker and @alyapunov: argreed to stick to the current codebase's approach. Fixed.